packages/design-system-react-native/src/temp-components/Spinner/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './Spinner';
+export type { SpinnerProps } from './Spinner.types';
```

Comment:
Here, we can export `IconColor` as `SpinnerColor` and update all instances of `IconColor` in `Spinner` to use `SpinnerColor`:

```suggestion
export type { SpinnerProps } from './Spinner.types';
export type { IconColor as SpinnerColor } from '../../components/Icon';
```